RELEASE/scripts/autoscend/auto_bedtime.ash

```diff
@@ -1199,7 +1199,7 @@ boolean doBedtime()
 			effect_to_wish = $effect[One Very Clear Eye];
 		}
 	}
-	if (auto_haveMonkeyPaw())
+	if (auto_canMonkeyPawWish())
 	{
```

Comment:
I don't think monkey paw combat skills are used, but things like the banish and olfact skill can't be used in G-Lover, while Monkey Paw wishing can be used.